eudemo/eudemo/ivc/plasma_face.py

```diff
@@ -91,7 +92,7 @@ def run(self) -> Tuple[BluemiraFace, BluemiraFace]:
         rm_clearance_face = _make_clearance_face(
             vessel_bbox.x_min,
             vessel_bbox.x_max,
-            self.wall_boundary.bounding_box.z_min,
+            self.wall_boundary.start_point().z,
```

Comment:
This may not be very robust, as it will rely on the orientation, or rather, direction of the wire. I appreciate bounding_boxes are also not particularly robust!

Reply:
The shape _should_ be an open wire, where both ends are equal in z. I've changed it to take the minimum of the start and end and added a comment.

Reply:
Both ends will be equal in z on in the case of a horizontal divertor splitting line. In future (read, near future) we're likely going to want to split the divertor from the blankets based e.g. on the angle of the lower port. This will undoubtedly result in both ends not having the same z value.